packages/docs/src/styles/_navigator.scss

```diff
@@ -13,16 +13,13 @@ $navigator-min-width: $pt-grid-size * 22;
     max-height: $navigator-height;
     overflow: auto;
   }
-}
 
-.pt-menu-item.pt-active {
-  background: $pt-intent-primary;
-  cursor: pointer;
-  color: $white;
+  .pt-menu-item:not(.pt-active) {
+    background: inherit;
+  }
 }
 
-.pt-menu-item.pt-active,
-.pt-menu-item:hover {
+.pt-menu-item.pt-active {
   .docs-result-path {
```

Comment:
merge this with line above so it's clearly one selector. `.pt-menu-item.pt-active .docs-result-path`